tasks/delete_app_objects_kind.yml

```diff
@@ -29,6 +29,6 @@
       - app={{ app.name }}
       - deployment_stamp={{ targeted_deployment_stamp }}
   register: remaining_objects
-  until: ( remaining_objects.resources | length ) == 0
+  until: remaining_objects.failed == True or ( remaining_objects.resources | length ) == 0
```

Comment:
Something more pythonic to ensure `.failed` is a boolean and not the result of a comparison would be:

```suggestion
  until: remaining_objects.failed is True or ( remaining_objects.resources | length ) == 0
```

Reply:
I would like to be a better :snake: , but It gives me the following error : 

```
fatal: [local]: FAILED! => {"msg": "The conditional check 'remaining_objects.failed is True or ( remaining_objects.resources | length ) == 0' failed. The error was: template error while templating string: no test named 'True'. String: {% if remaining_objects.failed is True or ( remaining_objects.resources | length ) == 0 %} True {% else %} False {% endif %}"}
```

![](https://media.giphy.com/media/jCOyMYDIe7Yd2/giphy.gif)

Reply:
Have you tried a lower case `true`?

Reply:
Yes, same error in lower case

Reply:
Maybe it is not Python but Jinja

Reply:
You are right: true is not a builtin test.
https://jinja.palletsprojects.com/en/2.10.x/templates/#list-of-builtin-tests

Reply:
Forget it.